code/game/machinery/doors/airlock.dm

```diff
@@ -38,6 +39,11 @@
 	var/bolt_up_sound = 'sound/machines/boltsup.ogg'
 	var/bolt_down_sound = 'sound/machines/boltsdown.ogg'
 
+/obj/machinery/door/airlock/New()
+	.=..()

```

Comment:
New() doesn't have a return value, may not want to confuse people by putting this here, or, better yet, put this in initialize() so it doesn't slow down map loads.

Reply:
Fix't and re-tested